gamedata/movedefs.lua

```diff
@@ -191,8 +191,8 @@ local moveDatas = {
 		badslope = 22,
 		badwaterslope = 255,
 		crushstrength = 25,
-		footprintx = 2,
-		footprintz = 2,
+		footprintx = 3,
+		footprintz = 3,
```

Comment:
Shouldn't this be called HOVER3 now? It seemed the name was intended to match the foorptinr size,